src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/InterpretedFrame.cs

```diff
@@ -1,20 +1,21 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Buffers;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Reflection;
 using System.Runtime.CompilerServices;
 
 namespace System.Linq.Expressions.Interpreter
 {
-    internal sealed class InterpretedFrame
+    internal unsafe struct InterpretedFrame
```

Comment:
Possibly, but it might need more changes. The struct is used as a field in a lambda quoter visitor class, for example. It does not have to, it could just keep constituent parts that it needs, but I did not want to make changes that I do not have to.